test/starlark_tests/targets_under_test/ios/BUILD

```diff
@@ -114,6 +114,24 @@ ios_application(
     ],
 )
 
+ios_application(
+    name = "app_with_custom_executable_name",
+    bundle_id = "com.google.example",
+    executable_name = "app.exe",
+    families = ["iphone"],
+    infoplists = [
+        "//test/starlark_tests/resources:Info.plist",
+    ],
+    minimum_os_version = "8.0",
+    tags = [
+        "manual",
+        "notap",
+    ],
+    deps = [
+        "//test/starlark_tests/resources:objc_main_lib",
+    ],
+)
```

Comment:
can you give this one a custom bundle_name too just to verify the order of operations prefers the custom `executable_name`